config/_default/menus/menus.es-us.yaml

```diff
@@ -0,0 +1,93 @@
+# Menus - Doks: https://getdoks.org/docs/recipes/menus/
+# Menus | Hugo: https://gohugo.io/content-management/menus/
+
+
+# header-bar/left side: main menus
+main:
+  - name: Descargas
+    url: /downloads/
+    weight: 10
+  - name: Docs
+    url: /docs/
+    weight: 20
+  - name: Rastreo de errores
+    url: https://github.com/Aegisub/Aegisub/issues
+    weight: 30
+  - name: Blog
+    url: /blog/
+    weight: 40
+
+# header-bar/right side: social menus
+social:
+  # - name: Twitter
+  #   pre: >-
+  #     <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0
+  #     0 24 24" fill="none" stroke="currentColor" stroke-width="2"
+  #     stroke-linecap="round" stroke-linejoin="round" class="feather
+  #     feather-twitter"><path d="M23 3a10.9 10.9 0 0 1-3.14 1.53 4.48 4.48 0 0
+  #     0-7.86 3v1A10.66 10.66 0 0 1 3 4s-4 9 5 13a11.64 11.64 0 0 1-7 2c9 5 20 0
+  #     20-11.5a4.5 4.5 0 0 0-.08-.83A7.72 7.72 0 0 0 23 3z"></path></svg>
+  #   url: 'https://twitter.com/aegisub'
+  #   weight: 10
+  - name: GitHub
+    pre: >-
+      <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0
+      0 24 24" fill="none" stroke="currentColor" stroke-width="2"
+      stroke-linecap="round" stroke-linejoin="round" class="feather
+      feather-github"><path d="M9 19c-5 1.5-5-2.5-7-3m14 6v-3.87a3.37 3.37 0 0
+      0-.94-2.61c3.14-.35 6.44-1.54 6.44-7A5.44 5.44 0 0 0 20 4.77 5.07 5.07 0 0
+      0 19.91 1S18.73.65 16 2.48a13.38 13.38 0 0 0-7 0C6.27.65 5.09 1 5.09
+      1A5.07 5.07 0 0 0 5 4.77a5.44 5.44 0 0 0-1.5 3.78c0 5.42 3.3 6.61 6.44
+      7A3.37 3.37 0 0 0 9 18.13V22"></path></svg>
+    url: 'https://github.com/Aegisub/Aegisub'
+    weight: 20
+
+
+# docs-sidebar/docs
+docs:
+  - name: Navegación
+    weight: 110
+    identifier: navigation
+  - name: Introducción
+    weight: 200
+    identifier: introduction
+  - name: Trabajando con subtítulos
+    weight: 300
+    identifier: working-with-subtitles
+  - name: Formatear
+    weight: 400
+    identifier: typesetting
+  - name: Sincronizar
+    weight: 500
+    identifier: timing
+  - name: Automatizar
+    weight: 600
+    identifier: automation
+  - name: Otro
+    weight: 700
+    identifier: miscellaneous
+  - name: Tutoriales-automatización
+    weight: 800
+    identifier: automation-tutorials
```

Comment:
I see you added a new sidebar menu since the Automation subpage isn't translated. Instead of adding this new menu item and having the reuglar automation one not working, should we just copy all untralsted pages? I think that is also currently the case for the handful of untranslated chinese pages.